client/src/services/APIservice.js

```diff
@@ -1,6 +1,6 @@
 import { API } from "../constants";
 
-export async function shrinkUrlService(token, longUrl) {
+export async function shrinkUrlService(token, longUrl, customCode) {
```

Comment:
What happens if we click on customise and no string is entered?

Reply:
Creates a link with randomly generated code 👍🏼